src/Microsoft.Extensions.Configuration.AzureAppConfiguration/TracingUtils.cs

```diff
@@ -75,6 +75,21 @@ public static bool IsDevEnvironment()
             return false;
         }
 
+        public static string GetAssemblyVersion(string assemblyName)
+        {
+            try
+            {
+                if (!string.IsNullOrEmpty(assemblyName))
+                {
+                    // Return the version using only the first 3 fields and remove additional characters
+                    return AppDomain.CurrentDomain.GetAssemblies()?.SingleOrDefault(assembly => assembly.GetName().Name == assemblyName)?.GetName().Version.ToString(3).Trim('{', '}');
+                }
+            }
+            catch (Exception exception) when (exception is AppDomainUnloadedException) { }
```

Comment:
```suggestion
            catch (AppDomainUnloadedException)
            {
                // Why is it ignored
            }
```

Reply:
Missed this from old changes, thanks

Reply:
Now that I looked into it more, I'm not really sure if this exception should be ignored, since it signifies a rare/critical error and sounds like it would affect other parts of the application. Maybe it makes sense to leave this catch out?